gateway/1.5/modules/ROOT/pages/flex-local-configuration-reference-guide.adoc

```diff
@@ -520,11 +603,22 @@ spec:
      password: ecomm-pwd-123
      DB: 7
 ----
+====
 
-==== Local Shared Storage Example
+.*Redis with TLS Shared Storage Example*
+[%collapsible]
+====
+The Redis storage service is an implementation of a REST API object store that functions as a bridge between the object store interface and a Redis backend. Optionally, you can use Redis with TLS to protect data and ensure secure communication.
+
```

Comment:
This is an internal implementation detail of Flex, I don't think customers should be aware of it